[rospy.client][INFO] 2020-01-19 17:03:55,539: init_node, name[/ros_test], pid[19782]
[xmlrpc][INFO] 2020-01-19 17:03:55,548: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-19 17:03:55,549: Started XML-RPC server [http://huai:45871/]
[rospy.init][INFO] 2020-01-19 17:03:55,549: ROS Slave URI: [http://huai:45871/]
[rospy.impl.masterslave][INFO] 2020-01-19 17:03:55,549: _ready: http://huai:45871/
[xmlrpc][INFO] 2020-01-19 17:03:55,549: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-01-19 17:03:55,550: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-01-19 17:03:55,649: registered with master
[rospy.rosout][INFO] 2020-01-19 17:03:55,649: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-19 17:03:55,651: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-19 17:03:55,651: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-19 17:03:55,943: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-19 17:03:58,046: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-19 17:05:04,144: topic[/camera/infra1/image_rect_raw] adding connection to [http://huai:34723/], count 0
[rospy.core][INFO] 2020-01-19 17:05:43,865: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-19 17:05:43,867: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-19 17:05:43,868: topic[/Lines2d] removing connection to /tracking_node
[rospy.internal][INFO] 2020-01-19 17:05:43,868: topic[/camera/infra1/image_rect_raw] removing connection to http://huai:34723/
[rospy.internal][WARNING] 2020-01-19 17:05:43,868: Unknown error initiating TCP/IP socket to huai:45655 (http://huai:34723/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-01-19 17:05:43,870: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 841, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2020-01-19 17:05:43,870: signal-2
[rospy.core][INFO] 2020-01-19 17:05:44,050: signal_shutdown [atexit]
